README.md

```diff
@@ -138,6 +140,36 @@ function debug(plugins) {
 }
 ```
 
+##### `encapsulateStyles`
+
+* Type: `Boolean`
+* Default: `false`
+
+Resets CSS properties to their default values to effectively allow a component
+to opt out of CSS inheritance and be encapsulated from the rest of the
+application. There are two types of CSS properties that affect components, inherited (e.g. `font-size,`
```

Comment:
Yep, mentioning Shadow DOM is good.

And regarding your second point I already added this section a bit lower:

> Only selectors conforming to the SUIT naming conventions are effected.